web/beacon-app/src/features/teams/components/RegisterNewUser/NewInviteRegistrationForm.tsx

```diff
@@ -106,46 +98,42 @@ function NewInviteRegistrationForm({ onSubmit, initialValues }: RegistrationForm
             {...getFieldProps('pwcheck')}
           />
         </div>
-        <CheckboxFieldset>
-          <Checkbox
-            name="terms_agreement"
-            onChange={(isSelected) => {
-              setFieldValue('terms_agreement', isSelected);
-              setFieldValue('privacy_agreement', isSelected);
-            }}
-            data-testid="terms_agreement"
-          >
-            I agree to the Rotational Labs Inc.{' '}
-            <Link to={EXTERNAL_LINKS.TERMS} className="font-bold underline" target="_blank">
-              Terms of Service
-            </Link>{' '}
-            and{' '}
-            <Link to={EXTERNAL_LINKS.PRIVACY} className="font-bold underline" target="_blank">
-              Privacy Policy
-            </Link>
-            .
-          </Checkbox>
-          <ErrorMessage component="p" name="terms_agreement" className="text-xs text-danger-500" />
-        </CheckboxFieldset>
         <Button
           type="submit"
           variant="secondary"
-          size="large"
           className="mt-4"
           isLoading={isSubmitting}
           disabled={isSubmitting}
           aria-label="Join Now"
         >
-          Join Now
+          <Trans>Join Now</Trans>
         </Button>
-        <p className="mt-2">No cost. No credit card required.</p>
+        <p className="mt-3">
+          <Trans>By continuing, you're agreeing to the Rotational Labs Inc.</Trans>{' '}
+          <Link
+            to={EXTERNAL_LINKS.TERMS}
+            className="font-bold text-[#1F4CED] underline"
+            target="_blank"
+          >
+            <Trans>Terms of Service</Trans>
+          </Link>{' '}
+          <Trans>and</Trans>{' '}
+          <Link
+            to={EXTERNAL_LINKS.PRIVACY}
+            className="font-bold text-[#1F4CED] underline"
+            target="_blank"
+          >
+            <Trans>Privacy Policy</Trans>
+          </Link>
+          .
+        </p>
       </Form>
     </FormikProvider>
   );
 }
 
 // TODO: Fix in the design system
-const CheckboxFieldset = styled.fieldset`
+/* const CheckboxFieldset = styled.fieldset */ `
```

Comment:
I kept the style for now in case we ever have to add a checkbox back to the form, but it can be removed.